src/utils/handleAndSignImages.ts

```diff
@@ -0,0 +1,32 @@
+import isSVGImage from '@/utils/isSVG';
+import { getFullSizeUrl } from '@/utils/getFullSizeUrl';
+import { maybeSignUri } from '@/handlers/imgix';
+import svgToPngIfNeeded from '@/handlers/svgs';
+import { getLowResUrl } from '@/utils/getLowResUrl';
+
+export function handleAndSignImages(
+  imageUrl?: string,
+  previewUrl?: string,
+  originalUrl?: string
+): {
+  imageUrl?: string;
+  lowResUrl?: string;
+} {
+  const image = imageUrl || originalUrl || previewUrl;
+
+  if (!image) {
+    return {};
+  }
+
+  const isSVG = isSVGImage(image);
```

Comment:
Is there some reason we need to check against a specific version of the image URL? We only check `imageUrl` in the old handler, so figured this was safe. CC @skylarbarrera